### ISSUE TITLE
Need help with a .bat File!

### ISSUE THREAD

**Issue by philw1508** [davidmann4/ogar-feeder-bot#387]
Hi i need some batch expertst.. ,

i want to make a loop function for startbot.bat  but when i do:
:loop
node f.js
goto loop

It doesnt work... even with i do it with a timeout. If i run it it's like if i only put node f.js in .bat file...
Need help very soon..
Thanks

**Comment by Nick8fr32** [comment]
i made one you just need to put in your feeder name
**@echo off**
**color 0a**
**echo Loop Mode**
**echo ---------**
**pause**
**cls**
**:start**
**cls**
**node //YOUR_FEEDER\.js**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo CRASH!!**
**echo ---------**
**cls**
**goto start**

**Comment by philw1508** [comment]
Ok will try

**Comment by philw1508** [comment]
Thanks

**Comment by Nick8fr32** [comment]
@philw1508 does it work?

**Comment by philw1508** [comment]
yes but i need a timeout after like 5 minutes and then it should restart

**Comment by Nick8fr32** [comment]
I Will try to make a button to restart the script

**Comment by philw1508** [comment]
Ok Thanks for ur reply :)

**Comment by Nick8fr32** [comment]
@philw1508 i have made the restart button just send ur email an i will send u the code

**Comment by philw1508** [comment]
Ok my email is : philip.wolf99@web.de  :)

**Comment by philw1508** [comment]
@Nick8fr32
